fastlane_bot/events/exchanges/solidly_v2.py

```diff
@@ -61,20 +56,15 @@ def get_events(self, contract: Contract) -> List[Type[Contract]]:
     async def get_fee(self, address: str, contract: Contract) -> Tuple[str, float]:
         if "velocimeter" in self.exchange_name:
             default_fee = await self.factory_contract.caller.getFee(address)
-            default_fee = float(default_fee)
-            if default_fee <= 1000:
-                default_fee = default_fee / 10000
-            else:
-                default_fee = default_fee / 10 ** 18
+            default_fee = float(default_fee) / 10000
```

Comment:
This doesn't reconcile with function `get_pool_fee`, which divides by `10 ** 18`

Reply:
Thank you - that was a leftover function from development that I forgot to remove. I've removed get_pool_fee and a few other unused functions.